.eslintrc.json

```diff
@@ -0,0 +1,60 @@
+{
+    "plugins": [
+      "prettier"
+    ],
+    "rules": {
+      "prettier/prettier": [
+        "error",
+        {
+          "singleQuote": true
+        }
+      ]
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:prettier/recommended"
+    ],
+    "env": {
```

Comment:
Should we add jQuery? It's still used in some scripts.

Reply:
I'll test the output of `yarn lint` on a file using jQuery and see if it's needed. :)